docs/source-2.0/additional-specs/smoke-tests.rst

```diff
@@ -0,0 +1,194 @@
+.. _smoke-tests:
+
+===========
+Smoke Tests
+===========
+
+Smoke tests are small, simple tests intended to uncover large issues by ensuring
+basic functionality works as expected. In Smithy, these tests are used to make
+sure clients can successfully make a request to a live service, and that the
+service responds as expected.
+
+
+--------
+Overview
+--------
+
+This specification defines a single trait in the ``smithy.test`` namespace that is
+used to make basic assertions about how the service should respond to a given
+request from the client.
```

Comment:
nit: omit "basic" and use "expectations" since that is what the trait uses

```suggestion
This specification defines a single trait in the ``smithy.test`` namespace that is
used to make expectations about how a service should respond to a given
request from a client.
```